pkg/tsdb/cloudwatch/time_series_query.go

```diff
@@ -98,6 +100,15 @@ func (e *cloudWatchExecutor) executeTimeSeriesQuery(ctx context.Context, req *ba
 	}
 
 	if err := eg.Wait(); err != nil {
+		var awsErr awserr.RequestFailure
+		if ok := errors.As(err, &awsErr); ok {
```

Comment:
It seems to me that we want to add the error to the dataframe in any case (e.g. a connection error), not only when the request fail. Are you thinking on some case in which we don't want to do that?

Reply:
Nah, I just wanted to unwrap them and I had something else on my mind before (it's been a while), I guess you're right.